src/styles/index.scss

```diff
@@ -1,3 +1,14 @@
+@import './utils/variables';
+@import './blocks/card';
```

Comment:
Ensure that the file `./blocks/card.scss` exists and is correctly referenced. If not, it will cause an import error.